Utils/Functions/Filesystem/Files/script.js

```diff
@@ -222,4 +222,19 @@ class Checksum {
             }
         });
     }
-}
+};
+
+module.ls = ls;
```

Comment:
Alternatively we could export the functions via:

```javascript
module.ls = function ls(...) {...};
```

I didn't do this here because some functions depend on each other which is problematic because if they are defined using `module.x = function x(...) {...};` the method also need to be accessed as `module.x` from inside the same script.